contrib/message-capture/message-capture-docs.md

```diff
@@ -0,0 +1,25 @@
+# Per-Peer Message Capture
+
+## Purpose
+
+This feature allows for message capture on a per-peer basis.  It answers the simple question: "Can I see what messages my node is sending and receiving?"
+
+## Usage and Functionality
+
+* Run `bitcoind` with the `-capturemessage` option.
```

Comment:
This should be `-capturemessages` (with ending "s") :)

Reply:
Thank you!